modules/040-terraform-manager/templates/terraform-state-exporter/deployment.yaml

```diff
@@ -104,6 +104,16 @@ spec:
         - "--v=2"
         - "--logtostderr=true"
         - "--stale-cache-interval=1h30m"
+        livenessProbe:
+            httpGet:
+              path: /livez
```

Comment:
As i see we expose /healthz endpoint
https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/operations/exporter.go#L208